### PR TITLE
Answers to Git Assignment Task 2

### PR DESCRIPTION
**Pull request by josephtv**

### CODE REVIEW

File: answers.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
+• What is a gitignore file and what should it include for a Java project?
+
+	A gitignore file is used to tell git to ignore certain files while tracking all the changes in our project directory.
+	These files won't be tracked by git and thus changes made to those files will be ignored.
+	While creating .gitignore file we can see that it includes file extensions for files which we do not want to track. 
+	
+	For a java project there are different files which we do not want to include in source control.
```
Comment (shanmugapriyan98) [inline answers.txt:7]: Can you please specify what kind of files will be ignored in java for better understanding ?

File: answers.txt [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	Thus differernt branch allows different users collaborating in the project to implement different features of the project without breaking the working master
+	branch. This allows developers and testers to test and verify that everything works before pushing their code to the master branch. This also helps
+	in resolving conflicts rather than everyone working in a single main branch. Thus different branches will ensure that one copy of our project will always run
+	in main branch.
+	
+-----------------------------------------------------------------------------------------------------------------------------------------------------------
+
+• In a project how many remote Git repositories do you usually have?
+
+	In a project we usually have one remote git repository. Two examples of remote git repositories are GitHub and BitBucket.
```
Comment (shanmugapriyan98) [inline answers.txt:42]: I think the question here is not type of git repository but the number of git repositories(github repos) we have ?

File: answers.txt (hunk 1 above)
Comment (josephtv) [reply answers.txt:7]: The gitignore file, as mentioned, includes compiled class files(.class), log files(.log), package files(.jar, .war, .zip), .project file, etc.

File: answers.txt (hunk 2 above)
Comment (josephtv) [reply answers.txt:42]: Repositories can be local and remote. Local repositories are the versions of our project which we have in our local machines. Whereas a remote repositories is a version of our project hosted on the network somewhere (like GitHub, BitBucket, etc.). 

In a project with only one repository, usually we may need only one remote git repository as everyone contributing in this project can be given access to this repository and they can have their own local versions of this repository where the can work individually and later all their work can be pushed to the remote repository for that particular project.

**Review by josephtv — COMMENTED** [review]
Added comments to review questions by @shanmugapriyan98

**Review by shanmugapriyan98 — APPROVED** [review]
Reviewed all the questions. Looks good. Approved.

**Pull request merged**